finance/models.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from candidate.models import SocialMediaModel, Person
+
+
+@python_2_unicode_compatible
+class Committee(SocialMediaModel):
+    COMMITTEE_TYPES = (
+        ('OF', 'Primarily-formed Official Committee'),
+        ('PF', 'Primariliy-formed Committee'),
+        ('IC', 'Independent Committee')
+    )
+    name = models.CharField(max_length=255)
+    address = models.ForeignKey('locality.Address', null=True)
+    locality = models.ForeignKey('locality.Locality', null=True)
+    committee_id = models.CharField(max_length=16, unique=True)
+    type = models.CharField(
+        max_length=2,
+        choices=COMMITTEE_TYPES,
+        help_text=''
+    )
+
+    def __str__(self):
+        return self.name
+
+
+@python_2_unicode_compatible
+class Company(SocialMediaModel):
+    name = models.CharField(max_length=255)
+    address = models.ForeignKey('locality.Address', null=True)
+    # locality = models.ForeignKey('locality.Locality', null=True)
+
+    def __str__(self):
+        return self.name
+
+
+@python_2_unicode_compatible
+class Form(models.Model):
+    """
+    """
+    name = models.CharField(max_length=255)
+    text_id = models.CharField(
+        max_length=32, help_text='e.g. 460 Schedule A')
+    FREQUENCY_TYPES = (
+        ('24', '24 hours'),
+        ('SA', 'Semi-annual'),
+        ('QU', 'Quarterly'),
+        ('OT', 'Other')
+    )
```

Comment:
>Can jurisdictions have different filing frequencies than listed? Just curious.

Not to my knowledge, after talking with @andrell81 and Stacey. I do think migrating later wouldn't be too troublesome, so I'm also not too worried about it. But if @andrell81 says otherwise, glad to design it in now.